fabric/src/test/java/TestRegex.java

```diff
@@ -350,6 +351,14 @@ public void GuildRankReplacementFeature_RECRUIT_USERNAME_PATTERN() {
         p.shouldMatch("§3[_user0name_");
     }
 
+    @Test
+    public void InfoBar_TERRITORY_INFO_PATTERN() {
+        PatternTester p = new PatternTester(InfoBar.class, "TERRITORY_INFO_PATTERN");
+        p.shouldMatch("§aLutho§2 [PROF]");
+        p.shouldMatch("§bCorkus City§3 [HOC]");
+        p.shouldMatch("§cDetlas§4 [AVO]");
```

Comment:
It seems form your changes to the regex that you got a match when you did not expect one. If so, please add a negative test case here that shows that your changes fix it.